src/instructlab/chat/chat.py

```diff
@@ -376,6 +376,7 @@ def start_prompt(self, logger, content=None, box=True):
                         **create_params,
                     )
                 except openai.BadRequestError as e:
+                    logger.debug("BadRequestError")
```

Comment:
```suggestion
                    logger.debug(f"BadRequestError: {e}")
```

Reply:
- linters do not like this approach. Use `logger.debug("BadRequestError: %s", e)` instead.
- You can also pass `exc_info=True` to any logging call in order to print the exception name, value, and traceback.